rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/identify/IdentifiedArchives.java

```diff
@@ -53,8 +53,7 @@ public static void addMappingsFrom(File file)
                     continue;
                 String[] parts = StringUtils.split(line, ' ');
                 if (parts.length < 2)
-                    throw new IllegalArgumentException("Expected 'SHA1 GROUP_ID:ARTIFACT_ID:VERSION[:COORDINATE]', but was [" + line + "] in ["
-                                + file + "]");
+                    throw new IllegalArgumentException("Expected 'SHA1 GROUP_ID:ARTIFACT_ID:[PACKAGING:[COORDINATE:]]VERSION', but was: \n" + line + "\n\tin [" + file + "]");
```

Comment:
I do find the Forge messages hard to read, but I think they would actually be harder as multiple line messages.